HCSocialSignIn/LandingViewController.swift

```diff
@@ -53,15 +55,13 @@ class LandingViewController: UIViewController {
         let redirectUrl = "http://hacknocraft.com/"
         let scope = ["r_basicprofile", "r_emailaddress"]
 
-        HCLinkedInManager.sharedInstance.login(viewController: self,
-                                               key: linkedInKey,
-                                               secret: linkedInSecret,
-                                               redirectUrl: redirectUrl,
-                                               scope: scope) { (success, _) in
-
-                                                if success {
-                                                    self.handleLinkedInLoginSuccess()
-                                                }
+        linkedInManager = HCLinkedInManager(key: linkedInKey, secret: linkedInSecret, redirectUrl: redirectUrl)
```

Comment:
Yeah, the `redirectURL` is useless for the user, we can define it as a constant.